### ISSUE TITLE
[Accessibility- Tab order C 12.8] Ensure consistent tab order

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#2941]
**Environment:** https://whitepaper-ft.meeds.io/

**Findings:**
Some pages don't provide a consistent tabbing path through content and functionality:

- Tabbing doesn't allow access to the content of the submenu of each page (the sub pages)

- Tabbing doesn't allow access to the features of modal windows (drawer) by clicking directly on the buttons that allow their opening: Exp the navigation site button

- Tabbing doesn't allow access to additional options on a page

- The "See More" button on "View All Spaces" does not handle focus resumptions: when activated, the focus remains on the button and the user is forced to tab back to reach the new displayed elements.

**Expected behavior:**

- Tabbing shouldn't reach visually hidden elements.

- When activating the "see more" button, the focus should be moved to the first of the newly displayed elements. It may be necessary to add a tabindex="-1" attribute if the element in question is not tabbable.

- When activating a button that opens a modal window, the focus should be moved to the first element of the window, and when closing it, the focus should return to the activation button

![Image](https://github.com/user-attachments/assets/d958fe3e-d32e-4ac3-aee7-af92224b0921)
